src/utils/media-utils.js

```diff
@@ -10,6 +10,7 @@ import { isIOS as detectIOS } from "./is-mobile";
 import Linkify from "linkify-it";
 import tlds from "tlds";
 import { mediaTypeFor } from "./media-type";
+import { MediaPlayer } from "dashjs";
```

Comment:
Since we are going to be re-testing this it might be a good time to update this dependency, its probably quite out of date.

Reply:
I want to test it in another PR (in next cycle?) because of it is at very last minute in this cycle now.